modules/giantsBidAdapter.js

```diff
@@ -0,0 +1,334 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER, NATIVE, VIDEO } from 'src/mediaTypes';
+import includes from 'core-js/library/fn/array/includes';
+
+const BIDDER_CODE = 'giants';
+const URL = '//d.admp.io/hb';
+const VIDEO_TARGETING = ['id', 'mimes', 'minduration', 'maxduration',
+  'startdelay', 'skippable', 'playback_method', 'frameworks'];
+const NATIVE_MAPPING = {
+  body: 'description',
+  cta: 'ctatext',
+  image: {
+    serverName: 'main_image',
+    requiredParams: { required: true },
+    minimumParams: { sizes: [{}] },
+  },
+  icon: {
+    serverName: 'icon',
+    requiredParams: { required: true },
+    minimumParams: { sizes: [{}] },
+  },
+  sponsoredBy: 'sponsored_by',
+};
+const SOURCE = 'pbjs';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['giants'],

```

Comment:
This is not needed.  The adapter is already called the `giants` adapter, so this alias is a duplicate.